content/posts/encoding-in-mei-hannahs-skotshne.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Encoding in MEI: \"Hannah's Skotshne\""
+date: 2025-06-02T16:39:54.646Z
+authors:
+  - Clara Byom
+description: "Encoding annotations, different readings, and more! (You don't
+  need to code to understand this blog post!) "
+tags:
+  - MEI
+categories:
+  - Klezmer
+---
+# Manuscript to Encoding
+
+Ah, our beloved "Hannah's Skotshne" (KMDMP 20-59-1676).
+
+By now, many of you have heard the tale of this tune that has swept around the globe since the launch of the Kiselgof-Makonovetsky Digital Manuscript Project (KMDMP). The original manuscript captured in the photo is torn, folded, and worn to the point of illegibility. 
+
+**Image 1: Facsimile of the "Hannah's Skotshne" Manuscript**
+
+![Image 1: Facsimile of the "Hannah's Skotshne" Manuscript](/images/uploads/kmdmp-20-59-1676-edited.png)
+
+Nevertheless, KMDMP digitizer extraordinaire Hannah Ochner tackled this piece, providing us with a legible version of the tune from the bits of writing peeking out from the folds and context throughout the tune. When Hannah was done, she submitted her MusicXML file for us to reference as we move into the editorial phase of encoding using MEI. 
+
+MEI is a XML specification (a version of the musicXML produced by music notation programs) developed by the Music Encoding Initiative used widely for digitally encoding music in academic and archival projects with similar goals to the Klezmer Archive. The schema is developed in the open and the project leaders welcome changes from the community. Indeed, we have already contributed a few small changes that have been accepted and will be included in future releases. The community also has built a suite of tools that we are excited to use, including the Verovio renderer and MEI-friend editor.
```

Comment:
To me, as written it sounds like you're saying that MEI is a version of MusicXML, which is not correct.
```suggestion
MEI is an XML specification (similar to MusicXML, which is produced by music notation programs) developed by the Music Encoding Initiative used widely for digitally encoding music in academic and archival projects with similar goals to the Klezmer Archive. The schema is developed in the open and the project leaders welcome changes from the community. Indeed, we have already contributed a few small changes that have been accepted and will be included in future releases. The community also has built a suite of tools that we are excited to use, including the Verovio renderer and MEI-friend editor.
```